spec/features/user_deletes_voter_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+feature 'user deletes a voter', %(
+As a signed up user
+I want to delete a voter
+So that I can remove ineligible or erroneous voters
+) do
+  # Acceptance Criteria
+  # [ ] From a voter's edit page, a user can click on a delete link
+  # [ ] If the user clicks this link and then confirms, the voter will be
+  #     deleted, and the user will be notified
+
+  scenario 'user deletes a voter' do
+    sign_in_as(FactoryGirl.create(:user))
+    voter = FactoryGirl.create(:voter)
+    visit edit_voter_path(voter)
+
+    click_on 'Delete this voter'
+
+    expect(page).to have_content('Voter deleted')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.